package/zoneinfo-utils/package

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+# Inspired by https://github.com/archlinux/svntogit-packages/blob/packages/tzdata/trunk/PKGBUILD
+
+pkgnames=(zoneinfo-utils)
+pkgdesc="Utilities for interacting with zoneinfo files"
+url=https://www.iana.org/time-zones
+pkgver=2020f-1
+timestamp=2020-05-04T06:16Z
+section=utils
+maintainer="Eeems <eeems@eeems.email>"
+license="custom: public domain"
+depends=()
+image=base:v1.2.1
+
+_tzcode=2020f
+_tzdata=2020f
```

Comment:
```suggestion
pkgnames=(zoneinfo-utils)
pkgdesc="Utilities for interacting with zoneinfo files"
url=https://www.iana.org/time-zones
_tzver=2020f
pkgver="$_tzver"-1
timestamp=2020-05-04T06:16Z
section=utils
maintainer="Eeems <eeems@eeems.email>"
license="custom: public domain"
depends=()
image=base:v1.2.1
```

And then use `$_tzver` instead of `$_tzcode` and `$_tzdata` below.